tests/selenium/popup_test.py

```diff
@@ -41,8 +41,11 @@ def switch_to_new_window(self, handles_before, max_retries=5):
 
         self.driver.switch_to.window(new_handle.pop())
 
-    def open_popup(self, close_overlay=True):
+    def open_popup(self, close_overlay=True, new_url=True):
         """Open popup and optionally close overlay."""
+        if new_url:
+            self.load_url("http://eff-tracker-site1-test.s3-website-us-west-2.amazonaws.com")
```

Comment:
Since `new_url` defaults to `True`, this opens http://eff-tracker-site1-test.s3-website-us-west-2.amazonaws.com for most popup tests. Seems like we should instead issue this instruction inside the one test that needs it. I'm also not clear on why we do it; we should explain with a comment.

Reply:
Don't we call `self.load_url` again directly below? Then don't we always end up on `self.popup_url`, regardless of the value of `new_url`?